Controller/FragmentAdminController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Controller;
+
+use Sonata\AdminBundle\Controller\CRUDController;
+use Symfony\Component\Form\Form;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * @author Hugo Briand <briand@ekino.com>
+ */
+class FragmentAdminController extends CRUDController
+{
+    /**
+     * @return Response
+     *
+     * @throws \RuntimeException
+     * @throws AccessDeniedException
+     */
+    public function formAction()
+    {
+        if (false === $this->admin->isGranted('CREATE')) {
```

Comment:
What about `if (!$this->admin->isGranted('CREATE'))` ? Why would `isGranted` not return a boolean?